cli/src/main/java/com/devonfw/tools/ide/process/ProcessContextImpl.java

```diff
@@ -360,11 +348,7 @@ private void performLogging(ProcessResult result, int exitCode, String interpret
 
   private void modifyArgumentsOnBackgroundProcess(ProcessMode processMode) {
 
-    if (processMode == ProcessMode.BACKGROUND) {
-      this.processBuilder.redirectOutput(Redirect.INHERIT).redirectError(Redirect.INHERIT);
-    } else if (processMode == ProcessMode.BACKGROUND_SILENT) {
-      this.processBuilder.redirectOutput(Redirect.DISCARD).redirectError(Redirect.DISCARD);
-    } else {
+    if (!processMode.isBackground()) {
```

Comment:
Do we still need this check if the only call of this method already checks that this has to be true?
Maybe replace with an assert instead.